.github/workflows/deploy-cparser-builder.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# Build and push docker image on changes to relevant files
+
+name: Deploy
+
+# Why deploy the builder image if it is supposed to be ephemeral? Because we
+# couldn't get the docker GitHub action to refer to a local-only image, even
+# with explicit build cache. Probably solvable, but this works as well, and it
+# does build the builder only once, so it removes duplication for only mildly
+# increased complexity.
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'scripts/**'
+    - 'docker/**'
+    # trigger on anything the follow-on deployments might need as well:
+    - 'seL4-platforms/**'
+    - 'cparser-run/**'
+    - 'preprocess/**'
```

Comment:
It might have been ok to leave these triggers in the downstream workflows, but this is fine.